src/app/dataValidator/helpers/dataLoader/readScenario.js

```diff
@@ -2,13 +2,13 @@ const fs = require('fs');
 const path = require('path');
 const { validateData } = require('../validators/validateData');
 const { countScenarios } = require('../../utilities/countScenarios');
-const readdirSync = require('./readdirSync');
+const asyncValidateFilesInDirectory = require('./asyncValidateFilesInDirectory');
```

Comment:
Can this be `const { asyncValidateFilesInDirectory }  = ` like the lines above it? Would tidy up line 11.

Reply:
I'm not fully able to explain way but I tried this previously be best practice reasons and I get the following error:
```
TypeError: asyncValidateFilesInDirectory is not a function
    at ifDirectoryThenValidateNestedFiles (/Users/phillee/Code/BBC/simorgh/src/app/dataValidator/helpers/dataLoader/readScenario.js:13:5)
    at Object.readScenario (/Users/phillee/Code/BBC/simorgh/src/app/dataValidator/helpers/dataLoader/readScenario.js:26:5)
...
```

Because of the above I changed to using `asyncValidateFilesInDirectory.asyncValidateFilesInDirectory()`